### ISSUE TITLE
Add an example to MLMMaskGenerator's docstring to show the best practice

### ISSUE THREAD

**Issue by chenmoneygithub** [keras-team/keras-nlp#120]
[MLMMaskGenerator](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/layers/preprocessing/mlm_mask_generator.py) is a helpful tool to generate masks for input sequences. The current docstring only has two basic examples, and it would be better for users if we can provide an example of how it should be used in a real MLM workflow. 

Basically the example will have:
1. Prepare data, we can generate random strings or load some small datasets from TFDS (tensorflow datasets).
2. Instantiate a tokenizer, and call tokenize on the data.
3. Apply `MLMMaskGenerator` on the tokenized data.
4. Feed the masked data to a dummy MLM model by calling `model.fit()`.

**Comment by adhadse** [comment]
I would like to take this issue. Please assignee me!

**Comment by chenmoneygithub** [comment]
Awesome, thanks!

**Comment by adhadse** [comment]
@chenmoneygithub Sorry for the late PR, got stuck in college work. I am stuck on dummy MLM model. Can you please help me out.

**Comment by chenmoneygithub** [comment]
@adhadse Definitely! What is the problem you see?

**Comment by adhadse** [comment]
The problem I am facing is most MLM model implementations I have come to see are on BERT, so what are we referring to dummy MLM model?
- Just input and output with no internal working?
- If it is for demo purpose should it be functional or class based model?

**Comment by chenmoneygithub** [comment]
Got it. Basically here the purpose is to showcase how the mask generator can fit in a workflow, so we just use any model that could run. In fact, the MLM is a bit complex because it requires some gathering. We can either use 1) pseudo-code that briefly summarizes how the training part likes. 2) [MLMHeader](https://github.com/keras-team/keras-nlp/blob/master/keras_nlp/layers/mlm_head.py) to show a real example.

**Comment by abuelnasr0** [comment]
I can take this.
